packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -201,9 +200,11 @@ function InserterBlockList( {
 	}, [ filterValue, items, categories, collections ] );
 
 	const childItems = useMemo( () => {
-		return filter( filteredItems, ( { name } ) =>
-			includes( rootChildBlocks, name )
-		);
+		return rootChildBlocks
+			? filter( filteredItems, ( { name } ) =>
+					rootChildBlocks.includes( name )
+			  )
```

Comment:
The whitespace here is wrong, but our Prettier config thinks it is correct. 😕 Here's what I think it should be:
```suggestion
				rootChildBlocks.includes( name )
			)
```

I've encountered this same issue a couple of times with ternary expressions involving multiple-line function calls.

Reply:
On the other hand, maybe the correct indentation would be:
```suggestion
					rootChildBlocks.includes( name )
				)
```
I'm not sure.